app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -279,7 +279,17 @@ def self.available_locks
 
       def import_package_profile(simple_packages)
         found = import_package_profile_in_bulk(simple_packages)
-        sync_package_associations(found.map(&:id).uniq)
+
+        # Build mapping: nvrea -> persistence
+        persistence_by_nvrea = simple_packages.each_with_object({}) do |sp, hash|
+          hash[sp.nvrea] = sp.persistence || 'unknown'
```

Comment:
Should `'unknown'` be a separate value? Or should we just assume `'persistent'` is the default?